### ISSUE TITLE
[a11y]: Element "button" should not be focusable within the subtree of an element with an 'aria-hidden' attribute with value 'true'

### ISSUE THREAD

**Issue by xiepingp** [carbon-design-system/carbon#11095]
### Package

carbon-components-react

### Browser

Chrome, Firefox

### Operating System

Windows, MacOS

### Package version

v10.39.0

### React version

v7.56.0

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker - Latest Deployment

### Description

When scanning the datatable with batch actions using Accessibility Checker, below violation is returned: Element "button" should not be focusable within the subtree of an element with an 'aria-hidden' attribute with value 'true'.
Same violations are happening when batch action bars is displayed (checkbox is selected).

### WCAG 2.1 Violation

IBM 1.3.1 Info and Relationships & IBM 4.1.2 Name, Role, Value

### CodeSandbox example

(not sure)

### Steps to reproduce

Scan the page using Accessibility Checker: https://react.carbondesignsystem.com/?path=/story/components-datatable-batch-actions--default

![Screen Shot 2022-03-28 at 3 41 04 PM](https://user-images.githubusercontent.com/63960906/160520195-6612c6a7-1cff-45c7-9d9d-51813cf6d70c.png)
![Screen Shot 2022-03-28 at 3 41 34 PM](https://user-images.githubusercontent.com/63960906/160520205-01982a52-b4fc-40b1-ac07-5ee0c3d7f69d.png)
![Screen Shot 2022-03-28 at 3 41 41 PM](https://user-images.githubusercontent.com/63960906/160520209-d0489ee8-b1f2-4e58-a4c1-7b1bbe804902.png)
![Screen Shot 2022-03-28 at 3 41 46 PM](https://user-images.githubusercontent.com/63960906/160520213-edf23f94-e872-467b-8701-9e4805ef0cf9.png)

Select the checkbox and the batch actions bar is displayed:
![Screen Shot 2022-03-29 at 3 25 00 PM](https://user-images.githubusercontent.com/63960906/160557392-2c8a4527-78d4-4a41-af17-f22ae24ff00e.png)
![Screen Shot 2022-03-29 at 3 25 47 PM](https://user-images.githubusercontent.com/63960906/160557405-0e3334e1-abda-44d7-b9e9-1948a23410b8.png)
![Screen Shot 2022-03-29 at 3 26 17 PM](https://user-images.githubusercontent.com/63960906/160557413-d2be0b87-9aa9-4d75-b64d-577427a96872.png)

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by tay1orjones** [comment]
My first thought is that this could be a false positive in that the decendants of the `aria-hidden` are not tabbable and have `tabIndex="-1"`, but it looks like this was a recent rule change https://github.com/act-rules/act-rules.github.io/issues/1807

There's a PR up to change the rule to no longer fail elements that are not tabbable https://github.com/act-rules/act-rules.github.io/pull/1820

**Comment by tay1orjones** [comment]
https://github.com/IBMa/equal-access/pull/796/

**Comment by tay1orjones** [comment]
IBMa has published an update to the `preview` archive used by the extension. You may need to restart the extension / browser, but you shouldn’t be seeing these with the ‘preview’ archive anymore.

**Comment by xiepingp** [comment]
Hi @tay1orjones , what do you mean of 'restart the extension / browser'?
I restart the extension 'IBM Equal Access Accessibility Checker' and restart the browse, the violations still exist:
![Screen Shot 2022-04-12 at 3 15 23 PM](https://user-images.githubusercontent.com/63960906/162903252-c6213b5e-6551-4194-a355-b7b43dd64f34.png)
Below is my extension setting:
![Screen Shot 2022-04-12 at 3 13 40 PM](https://user-images.githubusercontent.com/63960906/162903300-f7f3b605-72de-42e2-aaa2-9a714062179e.png)
